hw/ip/aes/doc/_index.md

```diff
@@ -157,12 +161,12 @@ For a general introduction into these cipher modes, refer to [Recommendation for
     1. The AES unit loads initial state into the State register inside the cipher core.
 
        _Depending on the cipher mode, the initial state is a combination of input data as well as IV._
-       _Note, if the CBC decryption is performed, or if running in CTR mode, the input data is also registered (Data In Prev in the block diagram)._
+       _Note, if CBC decryption is performed, or if running in CFB, OFB or CTR mode, the input data is also registered (Data In Prev in the block diagram)._
     2. The initial key is loaded into the Full Key register inside the cipher core.
 
        _Note, if the ECB/CBC decryption is performed, the Full Key register is loaded with the value stored in the Decryption Key register._
 
-    _Note, for the AES unit to automatically start in CBC/CTR mode, also the IV must be ready.
+    _Note, for the AES unit to automatically start in CBC, CBF, OFB or CTR mode, also the IV must be ready.
```

Comment:
CBF -> CFB

Reply:
Good catch, thanks. This will be fixed with #3060 .